graphql-schemas/schema.yml

```diff
@@ -4378,3 +4390,19 @@ confs:
   fields:
   - { name: name, type: string, isRequired: true }
   - { name: query, type: string, isRequired: true }
+
+- name: Network_v1
```

Comment:
Is there any field need to be unique? For example when I want to request a network, how do I know if there is a yaml file exist for this network already?

Reply:
I reckon the network address should be unique. I'll make that change.